grpc-rest.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency('grpc')
   spec.add_runtime_dependency('rails', '>= 6.0')
+  spec.add_runtime_dependency('gruf')
```

Comment:
`gruf` doesn't need to be a runtime dependency. I forgot that I didn't want to enforce this.

Reply:
Ah okay, sounds good. Just wondering, what is the rationale for not including it? I assumed that since we were explicitly using `Gruf::Interceptors::ServerInterceptor` in the code we'd need it as a dependency.

Reply:
That's only in the `send_gruf_request` method, which only gets used if Gruf is loaded.

Reply:
> Do you think adding the ErrorInterceptor you made by default into grpc-rest makes sense? Or is it specific to the publication stuff?

I feel like it might be too domain-specific to include in grpc-rest, but I am not sure. Maybe it would be best to include the error interceptor as an example in `README.md`? What do you think?

Reply:
Nah, I think the example we have is fine. I forgot it had some ActiveRecord etc stuff in there.